reports/pvnet_comp.md

```diff
@@ -0,0 +1,269 @@
+# OCFs PVNet Forecast Evaluation
+
+This report compares the accuracy of OCF’s national PV forecast (PVNet) and Sheffield Solar PVLive Intraday forecast, using the updated PVLive as the benchmark as well as additional analtysis on PVNets overall performance.
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+2. [Data Overview](#data-overview)
+3. [PVLive Intraday and PVNet vs PVLive Updated](#pvlive-intraday-and-pvnet-vs-pvlive-updated)
+   * [Results Table](#results-table)
+   * [Mean Absolute Error](#mean-absolute-error)
+   * [Mean Bias Error](#mean-bias-error)
+   * [Error Distribution](#error-distribution)
+4. [PVNet Model Evaluation](#pvnet-model-evaluation)
+   * [PVNet Results Table](#pvnet-results-table)
+   * [Mean Absolute Error](#mean-absolute-error-1)
+   * [Mean Bias Error](#mean-bias-error-1)
+5. [Conclusion](#conclusion)
+6. [Appendix](#appendix)
+
+## Introduction
+
+In this report, we evaluate the performance of OCFs national PV forecast (PVNet) against the Shefield Solar PVLive Intraday forecast, using the updated PVLive forecast as the benchmark truth for this comparison. This report will look into various error metrics and discuss the implications of our findings. First we will compare how PVLive Intraday and PVNet perform against PVLive Updated over periods in which we have data for. Then a more comprehensive evaluation of PVNet vs PVLive Updated is completed across the year of 2022, including evaluating the performance across different forecast horizons.
+
+This report should be used to indicate the difference in the forecasting capaboilities between the two as well as provide a framework for future analysis to completed over an larger number of data points.
+
+---
+
+## Data Overview
+
+The aim of these results is to evaluate the performance of the OCF PVNet model against the PVLive Intraday forecast over 2022. As PVLive Intraday is forecast with just a single horizon (at t=0 (now)), an equivlent horizon has been extracted from the PVNet forecast to make it a fair comparison.
+
+The number of datapoints that can be used to compare is restricted by two factors. Firstly, OCF started recording the data from PVLive intraday around halfway through 2022 (on 2022-06-14). Hence for the time preceeding this period we do not currently have the data available to compare against PVNet. Secondly, when creating the PVNet backtest there was some missing data, in paricular:
```

Comment:
The last sentence,  I'd change this to something like "Secondly, some input data which the PVNet model relies on was missing from our historic archive. In particular:"